models/attachment.go

```diff
@@ -75,29 +81,34 @@ func (a *Attachment) DownloadURL() string {
 func NewAttachment(attach *Attachment, buf []byte, file io.Reader) (_ *Attachment, err error) {
 	attach.UUID = gouuid.NewV4().String()
 
-	localPath := attach.LocalPath()
-	if err = os.MkdirAll(path.Dir(localPath), os.ModePerm); err != nil {
-		return nil, fmt.Errorf("MkdirAll: %v", err)
+	fs := storage.FileStorage{
```

Comment:
So we should remove method `LocalPath()` that we will not use that?

Reply:
Ah right, release part still consider attachment as local instead of file storage